h/static/scripts/directive/selection-tabs.js

```diff
@@ -7,21 +7,25 @@ module.exports = function () {
     bindToController: true,
     controllerAs: 'vm',
     //@ngInject
-    controller: function ($element, annotationUI) {
+    controller: function ($element, annotationUI, features) {
       this.TAB_ANNOTATIONS = uiConstants.TAB_ANNOTATIONS;
       this.TAB_NOTES = uiConstants.TAB_NOTES;
+      this.TAB_ORPHANS = uiConstants.TAB_ORPHANS;
 
       this.selectTab = function (type) {
         annotationUI.clearSelectedAnnotations();
         annotationUI.selectTab(type);
       };
+
+      this.orphansTabFlagEnabled = features.flagEnabled('orphans_tab');
```

Comment:
There is a potential issue here in that if the value of this feature flag changes, `orphansTabFlagEnabled` will not be updated. The easiest way to resolve this is to make it a function.